app/helpers/application_helper.rb

```diff
@@ -3,4 +3,20 @@ module ApplicationHelper
   def validate_params
     params[:donation][:amount].to_i > 0
   end
+
+  def bootstrap_class_for flash_type
+    { success: "alert-success", errors: "alert-danger", alert: "alert-warning", notice: "alert-info" }[flash_type.to_sym] || flash_type.to_s
+  end
+
+
+
+  def flash_messages(opts = {})
```

Comment:
Unused method argument - `opts`. If it's necessary, use `_` or `_opts` as an argument name to indicate that it won't be used. You can also write as `flash_messages(*)` if you want the method to accept any arguments but don't care about them.